app/models/credits/CreditsModel.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.credits
+
+import models.financialDetails.DocumentDetail
+import models.financialDetails.responses.ChargesResponse
+import play.api.libs.json.{Json, OFormat}
+
+case class CreditsModel(availableCredit: BigDecimal,
+                        allocatedCredit: BigDecimal,
+                        transactions: List[Transaction] )
```

Comment:
just curios if we can have a credit model without any transactions ?

Reply:
It's possible to have an empty list of transactions, if there are no credits / payments / refunds in the API response?